server/views/match/search.njk

```diff
@@ -5,12 +5,60 @@
 
 {% block content %}
 
-	<h1 class="govuk-heading-xl">Find a bed</h1>
+	<h1 class="govuk-heading-m">{{person.name}}</h1>
+	<dl class="search-identity-table govuk-body">
 
-	<h2 class="govuk-heading-m">Matching rooms: {{bedSearchResults.resultsRoomCount}}</h2>
-	<h2 class="govuk-heading-m">Matching premises: {{bedSearchResults.resultsPremisesCount}}</h2>
-	<h2 class="govuk-heading-m">Matching beds: {{bedSearchResults.resultsBedCount}}</h2>
+		<div class="search-identity-table__column">
+			<div class="search-identity-table__table-cell">
+				<dt>
+					<strong>Tier</strong>
+				</dt>
+				<dd>A1</dd>
+			</div>
+			<div class="search-identity-table__table-cell">
+				<dt>
+					<strong>CRN
+				</strong>
+				</dt>
+				<dd>{{person.crn}}</dd>
+			</div>
+			<div class="search-identity-table__table-cell">
+				<dt>
+					<strong>DOB
+				</strong>
+				</dt>
+				<dd>{{person.dateOfBirth}}</dd>
+			</div>
+			<div class="search-identity-table__table-cell">
+				<dt>
+					<strong>Expected arrival date</strong>
+				</dt>
+				<dd>{{expectedArrivalUi}}</dd>
+			</div>
+		</div>
+		<div class="search-identity-table__column search-identity-table__table-cell__last-col">
+			<div class="search-identity-table__table-cell">
+				<dt>
+					<strong>Assessment</strong>
 
+				</dt>
+				<dd>
+					<a href="/">View</a>
```

Comment:
We can use the `applicationId` and `assessmentId` from the placement request here now

Reply:
Also, for accessibility's sake, we should have some hidden text, ie:

```
<a href="">
  View <span class="govuk-visually-hidden">Assessment</span>
</a>
```